resources/views/maps/markers/edit.blade.php

```diff
@@ -60,6 +61,19 @@
                                 </a>
                             </li>
                         </ul>
+                    </div>  
+                    </div>  
+
+                    <div class="pull-left">
+                        <div class="btn-group">
+                                <a role="button" tabindex="-1" class="btn btn-dynamic-delete btn-danger" data-toggle="popover"
+                                    title="{{ __('crud.delete_modal.permanent') }}"
+                                    data-content="
+                                    <a href='#' class='btn btn-danger btn-block' data-toggle='delete-form' data-target='#delete-marker-confirm-form-{{ $model->id}}'>{{ __('crud.remove') }}</a>">
+                                    <i class="fa-solid fa-trash" aria-hidden="true"></i> {{ __('maps/markers.actions.remove') }}
+                                </a>
+                        </div>
+                        
                     </div>
                     @includeWhen(!request()->ajax(), 'partials.or_cancel')
```

Comment:
With the "or_cancel" view included here, it looks very ugly 
![image](https://user-images.githubusercontent.com/3263440/172020368-ec45cbe0-ab94-437f-b8ef-c6aac4cfd47a.png)

You can move it to the left of the "Save" button, and instead of using 
```php
@includeWhen(!request()->ajax(), 'partials.or_cancel')
```

use 
```php
@include('partials.footer_cancel')
```

Reply:
Changes implemented